KDICCore/KDICPlayer.swift

```diff
@@ -20,6 +20,7 @@ open class KDICPlayer {
     player.pause()
   }
   
+    /** to fix **/
```

Comment:
If you choose to do the documentation, make sure you reference the github issue number in the comment (#29).